test_runner/src/main/kotlin/ftl/config/common/CommonGcloudConfig.kt

```diff
@@ -162,6 +162,17 @@ data class CommonGcloudConfig @JsonIgnore constructor(
     @set:JsonProperty("type")
     var type: String? by data
 
+    @set:CommandLine.Option(
+        names = ["--fail-fast"],
+        description = [
+            "If true, only a single attempt at most will be made to run each execution/shard in " +
+                    "the matrix. Flaky test attempts are not affected. Normally, 2 or more" +
+                    " attempts are made if a potential infrastructure issue is detected. This" +
+                    " feature is for latency sensitive workloads."]
+    )
+    @set:JsonProperty("fail-fast")
+    var failFast: Boolean? by data
```

Comment:
should be documented in 
- `test_runner/flank.ios.yml`
- `/test_runner/flank.yml`
- `docs/index.md`

Reply:
Added